src/WebJobs.Script/Utility.cs

```diff
@@ -521,6 +521,13 @@ public static bool CheckAppOffline(string scriptPath)
             return false;
         }
 
+        public static string CleanUrl(string url)
```

Comment:
What is this for? Can you add a comment on the method explaining it?

Reply:
Added a comment. This is the same log helper we use in antares.

Reply:
In your implementation, if sig is the first query param (i.e. no &sig) you'll log the sig.

It's much simpler/safer to just not log ANY components from the query string, rather than mess with parsing. Doesn't something like the below give you what you're looking for, without us owning any uri parsing?

```csharp
public static bool TryCleanUrl(string url, out string cleaned)
{
    cleaned = null;

    Uri uri = null;
    if (Uri.TryCreate(url, UriKind.Absolute, out uri))
    {
        cleaned = $"{uri.Scheme}://{uri.Host}{uri.AbsolutePath}";
        if (uri.Query.Length > 0)
        {
            cleaned += "...";
        }
        return true;
    }

    return false;
}
```